src/js/modules/wallet/controllers/address-lookup/address-lookup.tpl.html

```diff
@@ -27,6 +27,16 @@
                 </div>
                 <span>{{ 'ADDRESS_LOOKUP_CHECKONLY_LABELED_SHORT' | translate }}</span>
             </label>
+
+            <label class="toggle-btc">
+                <div class="toggle">
+                    <input type="checkbox" ng-model="appControl.useOldAddress">
+                    <div class="track">
+                        <div class="handle"></div>
+                    </div>
+                </div>
+                <span>{{ 'ADDRESS_LOOKUP_FORMAT_BITCOIN_CASH' | translate }}</span>
```

Comment:
I can't remember, but I think boaz preferred a link instead of a checkbox here? Not sure if we were just looking at webwallet though